plugins/CoreHome/javascripts/notification.js

```diff
@@ -107,8 +107,21 @@
             $notificationNode.css(options.style);
         }
 
+        var defaultPlaceAt = '#notificationContainer';
+        var method = 'append';
+        if (options.placeat) {
+            defaultPlaceAt = options.placeat;
+        } else {
+            var modalSelector = '.modal.open .modal-content';
```

Comment:
If a modal is open, we want to make sure the error message is visible and therefore show it within the opened modal